### ISSUE TITLE
When exiting full screen, video stops being interactable (if it ended, it starts again)

### ISSUE THREAD

**Issue by GitThingsDan** [libre-tube/LibreTube#5215]
### Steps to reproduce

1. Click on video
2. Enter fullscreen
3. Exit fullscreen

### Expected behavior

Video should exit fullscreen without interruptions like loading again, or inconveniences like going back some seconds after loading.

Also, it shouldn't have 'visual glitches' like hiding the name of the channel, or stop being interactable.

### Actual behavior

Video loads a bit (which should not be necessary if  the video had that part already buffered), and then resumes, but it resumes some seconds before the current position.

Also (and maybe more important), it stops being interactable (it seems you still can double tap to pause, go back or forward, but no such icons appear on screen, which makes it very hard to do it), and it stops displaying the channel's name that's below the video. 

Finally, if one exits fullscreen when video has already ended, it starts playing again from the beginning (or, as it seems, from last video position that was saved in watch history). This one is very inconvenient when, for example, I just watched a music video and then exit fullscreen.

All in all, very strange behavior.

### LibreTube version

0.20.1

### Android version

Android 13

### Other details

https://github.com/libre-tube/LibreTube/assets/143112461/54770d1d-8d6c-4b9e-9433-525c5cdf52c4

_(I apologize for my English.)_

### Acknowledgements

- [x] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by GitThingsDan** [comment]
It seems I didn't check well enough for similar older issues...could this one recently closed (#5162) be related?

**Comment by akano12** [comment]
Try and report back with nightly builds

**Comment by GitThingsDan** [comment]
> Try and report back with nightly builds

LibreTube 0.20.1 Debug (default settings): everything seems to be OK here
https://github.com/libre-tube/LibreTube/assets/143112461/867f1409-d532-44e6-9fc3-d28f936e0daf

LibreTube 0.20.1 Debug (my settings): 
https://github.com/libre-tube/LibreTube/assets/143112461/bd313426-84f7-4548-86d0-c734a7e44159

According to the screen recordings above, it looks like the problem is within my settings: 
[libretube-backup-2023-11-22-22_59_48.json](https://github.com/libre-tube/LibreTube/files/13445507/libretube-backup-2023-11-22-22_59_48.json)

**Comment by TryingToUseLinux** [comment]
The setting that is causing this is screen rotation: auto rotate.
It is occuring during switching between landscape and portrait orientation even when you don't enter full screen mode. 
The bug occurs in v0.20.0 and 0.20.1, as they have the new landscape player layout.

**Comment by Bnyro** [comment]
That the player controls are not responsive is a duplicate of/the same issue as https://github.com/libre-tube/LibreTube/issues/4161. Please append your findings related to it there.

For the other issue, please test this again with a debug build after #5635. This doesn't fix the issue 100%, but it should already prevent the player from reloading at least.
